src/test/unit/pid_unittest.cc

```diff
@@ -529,12 +529,12 @@ TEST(pidControllerTest, testItermRelax) {
 
     applyItermRelax(FD_PITCH, pidData[FD_PITCH].I, gyroRate, &itermErrorRate, &currentPidSetpoint);
 
-    ASSERT_NEAR(-6.66, itermErrorRate, calculateTolerance(-6.66));
+    ASSERT_NEAR(-8.16, itermErrorRate, calculateTolerance(-6.66));
```

Comment:
This should have been
ASSERT_NEAR(-8.16, itermErrorRate, calculateTolerance(-8.16));